### ISSUE TITLE
Broken UI on web browsers: Chrome & Opera

### ISSUE THREAD

**Issue by michalslowikowski00** [apache/kibble-1#71]
**Description:**
UI is broken on Chrome and Opera.
UI on Safari and Firefox is ok.

**Reproduction steps:**
1. Run Kibble:
- pip install -e ."[devel]"
- docker-compose -f docker-compose-dev.yaml up setup
- docker-compose -f docker-compose-dev.yaml up ui
2. Go to http://127.0.0.1:8000/ in Chrome
3. Go to http://127.0.0.1:8000/ in Opera

**Actual result:**
UI is broken and practically unusable.

**OS:**
MacOS Catalina

**Logs:**
-

**Other:**
Screenshot from Chrome (on Opera is the same result)
<img width="757" alt="Screen Shot 2020-10-24 at 18 07 21" src="https://user-images.githubusercontent.com/22084125/97086621-c3877400-1624-11eb-95ed-f0dc23fe2c95.png">

**Comment by turbaszek** [comment]
Thanks @michalslowikowski00 for reporting this, do I correctly understand that this issue impacts only the two mentioned browsers?

**Comment by michalslowikowski00** [comment]
Yes. Chrome and Opera. Opera is a minor issue in my opinion but Chrome is a big thing.
